go/acct/radius.go

```diff
@@ -141,7 +141,7 @@ func (h *PfAcct) sendRadiusAccounting(r *radius.Request) {
 		"X-FreeRADIUS-Section": "accounting",
 	}
 
-	if err := h.AAAClient.Notify("radius_accounting", attr, 1); err != nil {
+	if _, err := h.AAAClient.Call("radius_accounting", attr, 1); err != nil {
```

Comment:
Notify or Call ?

Reply:
Call for now

Reply:
Better to call for now since otherwise pfacct accepts too many concurrent requests which can bring it to a halt